cache-filter/src/filter/http.rs

```diff
@@ -367,26 +375,42 @@ impl CacheFilter {
             upstream: upstream_builder.build(&cluster_name, timeout),
         })
     }
+
+    #[cfg(feature = "visible_logs")]
+    #[allow(dead_code)]
+    fn on_http_response_header(&mut self, _: usize) -> Action {
```

Comment:
It's weird that this is called out as dead code. Have you tested that this is indeed being executed?

Reply:
Haven't really tested it out but Clippy thinks it is dead code.